Mathlib/Tactic/NormNum/BigOperators.lean

```diff
@@ -109,7 +109,7 @@ set_option linter.unusedVariables false in
 
 Fails if we cannot determine which of the alternatives apply to the expression.
 -/
-partial def List.proveNilOrCons {α : Q(Type u)} (s : Q(List $α)) :
+partial def List.proveNilOrCons {u : Level} {α : Q(Type u)} (s : Q(List $α)) :
```

Comment:
Any reason for having this both here, and at the top of the file?

Reply:
If I remove this, line 142 throws an error
```
application type mismatch
  ?m.5768 v head✝¹ fst✝ β head✝ f xxs this
argument
  this
has type
  ?_pp_uniq.9 =Q ?_pp_uniq.10 : Prop
but is expected to have type
  ?_pp_uniq.12 =Q ?_pp_uniq.13 : PropLean 4